internal/server/statvar/golden/get_statvar_group_node/demographics_gbr.json

```diff
@@ -103,7 +103,7 @@
       "id": "dc/g/Person_Age",
       "specializedEntity": "Age",
       "displayName": "Person by Age",
-      "descendentStatVarCount": 663
+      "descendentStatVarCount": 666
```

Comment:
i don't recall new data that would affect GBR. is this from a hierarchy cleanup?

Reply:
The same 3 SVs I believe...